src/transaction/transaction.ts

```diff
@@ -207,10 +207,11 @@ export const sendTransaction = async function (
 
   let txnToSend = transaction
 
+  const resroucePopEnabled = sendParams?.populateAppCallResources ?? Config.populateAppCallResources
```

Comment:
Misspelt variable. Suggest `populateResources`?

Reply:
Done in c07065e48b842f59aec1102add565809144dada4